_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Hearing with light: photoacoustic radiology improved"
+author: RaoOfPhysics
+---
+
+_Using interstitial light source for tissue-imaging technique based on photoacoustic spectroscopy improves penetration depth and shows promise for minimally invasive radiology_
+
+Photoacoustics (PA) involves shining modulated light (*photo–*) on a material, causing the substance to undergo rapid thermal expansion and contraction in such a way that the pressure differences manifest as sound waves (*acoustics*)[^1]. Materials vary in the sounds they produce corresponding to light at different wavelengths. A light-vs-sound spectrum for assorted materials can be created by “listening” to the pressure waves produced in that material by light shone on it at varying wavelengths. The phenomenon was discovered by Alexander Graham Bell in the late 19th Century, but has found use in modern materials science relatively recently.
+
+PA imaging provides a guiding mechanism for medical procedures without the risks of ionising radiation and the cost of expensive equipment. Light produced by lasers is shone on photoabsorbers, such as haemoglobin, which produce pressure waves that are detected by transducers to produce images of the local region. PA also provides high-contrast images of the surgical tools being used in the medical procedure.
+
+However, this imaging technique has come with a severe limitation thus far: the deeper the light goes into tissue, the less photo-energy per unit area the material receives, thus reducing the resolution and constraining PA imaging’s penetration depth. Since simply increasing the energy of the laser is out of the question &mdash; these matters being regulated by medical bodies such as the American National Standards Institute &mdash; a new approach has recently been explored. In laboratory (non-clinical) conditions, researchers modified their apparatus so that a fiberoptic light source was inserted into the material under observation allowing the light to be shone not from the surface but from within the material (i.e. it is irradiated *interstitially*). The transducers, responsible for picking up the sound waves, were located externally themselves. Tests were carried out on a laboratory surrogate for tissue known as a tissue-mimicking phantom as well as prostate tissue from pigs and liver tissue from cows.
+
+The researchers found that the use of an interstitial light source improved the penetration depth of the imagine technique well beyond previously established limits. Wires inserted into the tissue surrogate at depths of 7, 20, 37 and 47 mm were clearly visible using the novel method, whereas the wires were only seen at the first two depths when the conventional method was used. Overall, the researchers were able to demonstrate that the method of interstitial irradiation provides a much higher penetration depth for PA imaging.
```

Comment:
I added a point about the source to the previous paragraph (it's made a part of the surgical apparatus).